src/code/circuit_python/gizmo.md

```diff
@@ -0,0 +1,163 @@
+# The Circuit Python Gizmo Module
+
+In this tutorial, we will introduce the `circuitpython_gizmo` module and
+the `print` function. You will learn how to get input from your gamepad
+and how to send text back to your computer to help debug your code.
+
+## The print Function
+
+While your Circuit Python programs are running while your Gizmo is
+connected to a computer, they are able to send information back to your
+computer. The messages your Gizmo sends will show up in the "serial"
+viewer in the Mu editor. Your code sends these messages by calling the
+`print` function.
+
+> [!NOTE]
+>
+> "Serial" is a generic term that refers to several different low-level
+communication protocols used in electronics. Often this refers to
+[UART](https://en.wikipedia.org/wiki/Universal_asynchronous_receiver-transmitter)
+used to communicate between microcontrollers. For our purposes here, the
+USB connection is emulating a simple "serial" communication protocol.
+
+To use the `print` function, give it the message you want to send. This
+can be text, numbers, or variables. The `print` function is always
+available, so you don't have to import any modules to use it.
+
+```Python
+# Sending text
+print("Hello from Gizmo!")
+
+# Sending variables
+value = 100
+print(value)
+```
+
+You can also embed variables into text messages to give more context. Just
+put an 'f' in front of your message and wrap the variable names in curly
+brackets. For example:
+
+```Python
```

Comment:
This should probably have a note block near it to call out that you just introduced the named concept of format strings, with an appropriate link to documentation on format strings.